askama_parser/src/node.rs

```diff
@@ -577,7 +577,7 @@ impl<'a> FilterBlock<'a> {
                 ws(identifier),
                 opt(|i| Expr::arguments(i, s.level.get(), false)),
                 many0(|i| filter(i, s.level.get())),
-                opt(Whitespace::parse),
+                ws(opt(Whitespace::parse)),
```

Comment:
Wouldn't this allow `{% filter lower|indent(2) - %}`, too?

Reply:
Good catch! I'll add a (ui) test to ensure it doesn't happen by error.